adds/start.sh

```diff
@@ -58,7 +58,13 @@ case $OPTION in
 	"start")
 		if [ -f /data/turnserver.conf ]; then
 			echo "-=> start turn"
-			/usr/bin/turnserver --daemon -c /data/turnserver.conf
+			if [ -f /conf/turnserver.conf.deactivated ]; then
+				mv -f /conf/turnserver.conf.deactivated /conf/turnserver.conf
+			fi
+		else
+			if [ -f /conf/turnserver.conf ]; then
+				mv -f /conf/turnserver.conf /conf/turnserver.conf.deactivated
+			fi
```

Comment:
`turnserver.conf.deactivated`?
You probably mixed the conf files of supervisor and turnserver?